M/Mineos/build_tarballs.jl

```diff
@@ -3,19 +3,37 @@
 using BinaryBuilder
 
 name = "Mineos"
-version = v"1.0"
+version = v"1.0.2" 
 
 # Collection of sources required to build Mineos
 sources = [
-    "https://github.com/anowacki/mineos.git" =>
-    "e2558b486d7656ef112608a8776643da66dc87cf",
+    GitSource("https://github.com/geodynamics/mineos.git", "3dd7c7433766d630b929d8254d03e705808ff8a3"),
+    DirectorySource("./bundled"),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
 cd ${WORKSPACE}/srcdir/mineos
-./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} --disable-doc
+
+# Drop docs build target
+atomic_patch -p1 ../patches/drop_docs.diff
+
+autoupdate
```

Comment:
I believe (per my comment in the issue mentioned below) that with autotools it is more usual to create the configure script before distributing the source, and that the build step (i.e., what's happening here) should not do that.  Hence my use of my own fork of the repo.

However, I may not be up to date on this.  @giordano is it customary on here to rerun the autotools commands to build the configure script?

Reply:
There are different opinions about checking the `configure` script in a repository: since that's an automatically generated file, most projects do _not_ check it in, but then they provide release tarballs which have been bootstrapped by the package maintainer.  However this means that if you try to do a build directly from a repository checkout you need to bootstrap yourself, which sometimes may be complicated.

Reply:
Think in this case, it makes sense to use the fork, particularly if @anowacki feels some level of responsibility to continue maintaining the project. :)

Reply:
(It also lets us drop the patch and use the build flag)